qiskit/circuit/quantumcircuit.py

```diff
@@ -1262,31 +1263,46 @@ def parameters(self):
         """Convenience function to get the parameters defined in the parameter table."""
         return set(self._parameter_table.keys())
 
-    def bind_parameters(self, value_dict):
+    @property
+    def num_parameters(self):
+        """Convenience function to get the number of parameter objects in the circuit."""
+        return len(self.parameters)
+
+    def bind_parameters(self, value_dict, in_place=False):
```

Comment:
```suggestion
    def bind_parameters(self, value_dict, inplace=False):
```
For consistency with `inplace` elsewhere in terra.